### ISSUE TITLE
`CoilSet` "Family" akin to `EquilibriaFamily`

### ISSUE THREAD

**Issue by ddudt** [PlasmaControl/DESC#1072]
I/O wrapper for a list of coil sets. Could be useful for keeping track of coil sets that correspond to equilibria within a family.

**Comment by f0uriest** [comment]
Might be more useful as a general IOAble family for arbitrary things, not just coils. And could make EquilibriaFamily a subclass

**Comment by dpanici** [comment]
define a `save` function in `desc.io` and allow saving of lists of `IOAble` objects

**Comment by ddudt** [comment]
I would also be in favor of removing the `EquilibriaFamily` class, since it has no unique functionality other than `solve_continuation`. Even then I think `solve_continuation_automatic` is more commonly used and in that case the user typically doesn't care about the intermediate solutions.

**Comment by dpanici** [comment]
I'd be in favor of that as well

**Comment by f0uriest** [comment]
I don't think we should get rid of it, since that would likely break every existing saved `EquilibriaFamily`
